.eleventy.js

```diff
@@ -60,7 +64,7 @@ module.exports = function (config) {
   return {
     passthroughFileCopy: true,
     dir: {
-      input: 'src',
+      input: hasNotes ? 'src/{index.njk,notes.njk}' : 'src/!(notes.njk)',
```

Comment:
It appears that the generated index.html does not include any slides (the `main` element is empty) ...

Reply:
Mmh indeed. It seems like its either we only build `index.html` file and it will be empty or we build `index.html` and every slide will be rendered as a separate page.

Will investigate but the second option seems the only working one 👍

Reply:
Mmh it seems this glob is okay ([as tested here](https://globster.xyz/?q=src%2F%7Bindex.njk%2Cnotes.njk%2Cslides%2F!(*.notes).md%7D&f=src%2Findex.njk%2Csrc%2Fnotes.njk%2Csrc%2Fslides%2Florem.md%2Csrc%2Fslides%2Florem.notes.md%2Csrc%2Fslides%2Fipsum.md%2Csrc%2Fslides%2Fipsum.notes.md)) but it seems not valid for Eleventy.

```
src/{index.njk,notes.njk,slides/!(*.notes).md}
```

I can check tomorrow if I have time to fix it, otherwise I'll do it after my holidays! ⛰️